server/src/main/java/org/elasticsearch/indices/recovery/MultiFileWriter.java

```diff
@@ -118,7 +118,7 @@ public void writeFile(StoreFileMetadata fileMetadata, long readSnapshotFileBuffe
             }
 
             Store.verify(indexOutput);
```

Comment:
Can we also only do this when `verifyOutput` is `true`?

Reply:
++ done